doc/_templates/layout.html

```diff
@@ -2,7 +2,7 @@
 {% block document %}
   {% if is_release %}
     <div class="wy-alert wy-alert-danger">
-      The <a href="/latest/{{ pagename }}.html">latest development version</a>
+      The <a href="/latest/{{ pagename }}.html" rel="canonical">latest development version</a>
```

Comment:
Is this really a correct way to specify `rel="canonical"`?

Various sources, including the [original RFC](https://www.ietf.org/rfc/rfc6596.txt), only mention the `link` tag (in `head`) as the place where this attribute should be specified.